voice-output/voice-pack-voicer.js

```diff
@@ -23,9 +23,35 @@ function weightedRandom(weights) {
 }
 
 class VoicePack {
-  constructor(syllableFiles, audioBuffer) {
+  constructor(syllableFiles, actionFiles, voiceActor, audioBuffer) {
     this.syllableFiles = syllableFiles;
+    this.actionFiles = actionFiles;
     this.audioBuffer = audioBuffer;
+    this.voiceActor = voiceActor;
+    this.actionVoices = actionFiles.map(({name, offset, duration}) => {
+      return {
+        name,
+        offset,
+        duration,
+        nonce: 0,
+      };
+    });
+  }
+  selectVoice(voicer) {
+    // the weight of each voice is proportional to the inverse of the number of times it has been used
```

Comment:
This is copy paste, so it seems like this algorithm should be a generic util.

Reply:
move `selectVoice` into util.js.

Reply:
That's good, but then why is it still here?

Reply:
Sorry, I’m not sure what do you mean. You mean selectVoice still in voice-pack-voicer?

Reply:
Yes, it's fixed now, it just wasn't when I was making the comment.